### DISCUSSION TITLE
Storage - GIT - Is this ment to be inside commit...???

### DISCUSSION THREAD

**Discussion by Zappo-II** [requarks/wiki#4126] [Help / Questions]
Hi There, did not find any information about what is supposed to be inside the git commit if I use git as a storage repository...

* I use the latest docker image, enabled git and made shure it checks green...
* Since there already are some pages created (we are doing an evaluation) I pressed the "Untracked Changes" button...
* After that it takes a VERY LONG TIME (and the docker logs shows a lot of action) and the jswiki interface is not reachable...
* After that long time, the repo contains some (very few) files (I tend to expect there) and directory `home` (I would also expect) and a directory `app/wiki` containing things that (apear to be "code" and) I would not expect to be in there...

Question:

* "Is that supposed to be like that...???"
* "What went wrong / what did I do wrong...???"

My Local Repository Path is (default I think) `./data/repo`

Insights welcome...
BRG...

**Comment by NGPixel** [comment]
Check the logs of your docker container for errors.

**Reply by Zappo-II** [reply]
THX 4 response but not helpful at all...

Let me rephrase the question...

If I have created jswiki content BEFORE having set up git storage and then configure git storage and press the "Untracked Changes" button in the storage configuration section, what is supposed to show up inside `./data/repo/` ...???

**Reply by NGPixel** [reply]
"Add Untracked Changes" will output all pages from the DB into the files in the repo path you entered. From there you can force a sync to push these files to your remote git.
